src/WebUI/Resources.js

```diff
@@ -84,4 +84,5 @@ define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], fu
     exports.LayersText = "Layers";
     exports.CommandText = "Command";
     exports.SizeText = "Size";
+    exports.SummaryHeaderSubTextFormat = "{0} (cluster)";

```

Comment:
remove "(cluster)" from here, so no need to have this variable

Reply:
Leaving this for the time as this is the way it is in the figma. Let us have a UI review with PMs and designers and we'll incorporate the feedback